.github/workflows/package-tests.yml

```diff
@@ -37,8 +37,8 @@ jobs:
           export PROJECT_VERSION="${{ github.event.inputs.project_version }}"
           echo "project_version: ${{ github.event.inputs.project_version }}"
           # To be able to test pipeline without triggering with project_version parameter using workflow_dispatch parameter,
-          # if workflow_dispatch parameter is empty, 10.2.1 parameter is set to execute pipeline.
-          [ -z ${PROJECT_VERSION} ] && export PROJECT_VERSION=10.2.1
+          # if workflow_dispatch parameter is empty, 11.1.4 parameter is set to execute pipeline.
```

Comment:
10.2.1 does not have ubuntu jammy packages, so I updated into 11.1.4 for default tests